### ISSUE TITLE
[AllyAudit] Confirm Color Contrast

### ISSUE THREAD

**Issue by ssabo1** [sparkdesignsystem/Backlog#1327]
**User Story**
As Spark we want to confirm color contrast of our foreground and background colors so that it doesn't affect usability.

**Notes:**
Paciello Audit on Spark Design System
QLC-3
Drop down, Masthead and Inputs
Insufficient text color contrast

ISSUE DESCRIPTION
The level of contrast between foreground and background colors affects usability. More contrast makes it easier to distinguish characters and symbols. It is therefore important to ensure that there is sufficient contrast between the color of text and its background color, so that is easily readable. This is particularly important for people with visual impairments that affect visual acuity or color perception.

WCAG requires a minimum contrast ratio of 4.5:1 for "regular" text (smaller than 18pt / 24px, or bold and smaller than 14pt / 18.5px) and a minimum of 3:1 for large text (at least 18pt / 24px, or bold and at least 14pt / 18.5px).

We encountered a number of color contrast issues where content lacks sufficient contrast. Typical examples are shown below.

RECOMMENDATIONS
Provide enough contrast between the foreground (text) color and background color so that people who are color-blind or with moderately low vision can read text and distinguish elements. For small text (less than 18pt / 24px, or less than 14pt / 18.667px for bold text), the contrast between the color of the text and the color of its background should be at least 4.5:1. For large text (at least 18pt / 24px, or at least 14pt / 18.667px for bold text), the contrast ratio should be at least 3:1.

**AC**

**Comment by ssabo1** [comment]
Confirm, no longer an issue since color update?

**Comment by ssabo1** [comment]
This is covered with our Spark Color Page.
